linkerd/proxy/http/src/detect.rs

```diff
@@ -92,6 +92,7 @@ where
     H: NewService<(HttpVersion, T)>,
 {
     pub fn new(target: T, server: Server, new_http: H, new_tcp: F, drain: drain::Watch) -> Self {
+        tracing::trace!("new AcceptHttp");
```

Comment:
we _probably_ don't need this, i stuck it in while debugging and forgot to take it out
```suggestion
```